gravitee-apim-cypress/cypress/integration/platform/jwt-policy-tests.ts

```diff
@@ -0,0 +1,320 @@
+        .ok()
+        .its('body')
+        .then((api) => {
+          rs256Api = api;
+          publishPlan(API_PUBLISHER_USER, rs256Api.id, rs256Api.plans[0].id).ok();
+        })
+        .then(() => {
+          deployApi(API_PUBLISHER_USER, rs256Api.id).ok();
+          startApi(API_PUBLISHER_USER, rs256Api.id).noContent();
+        });
+    });
+
+    cy.log("-----  Create an API that doesn't propagate Auth header  -----");
+    const jwtApiImport_noAuthProp = ApiFakers.jwtApi({ resolverParameter: secret, propagateAuthHeader: false });
+    jwtApiImport_noAuthProp.proxy.groups[0].endpoints[0].target = 'https://api.gravitee.io/echo';
+    importCreateApi(API_PUBLISHER_USER, jwtApiImport_noAuthProp)
+      .ok()
+      .its('body')
+      .then((api) => {
+        noAuthPropApi = api;
+        publishPlan(API_PUBLISHER_USER, noAuthPropApi.id, noAuthPropApi.plans[0].id).ok();
+      })
+      .then(() => {
+        deployApi(API_PUBLISHER_USER, noAuthPropApi.id).ok();
+        startApi(API_PUBLISHER_USER, noAuthPropApi.id).noContent();
+      });
+
+    cy.log('-----  Create an application for JWKS test in AM  -----');
+    const am_domainName = `${faker.random.word()}${faker.datatype.number()}-Domain`.toLowerCase();
+    const am_jwksApplicationName = `noScope-application_${faker.datatype.number()}`;
+
+    am_getApiToken(AM_ADMIN_USER)
+      .then((response) => (am_apiToken = response.body.access_token))
+      .then(() => am_createDomain(am_apiToken, am_domainName))
+      .created()
+      .then((response) => {
+        am_domainId = response.body.id;
+        am_domainHrid = response.body.hrid;
+      })
+      .then(() => am_enableDomain(am_apiToken, am_domainId))
+      .ok()
+      .then(() => {
+        am_createApplication(am_apiToken, am_jwksApplicationName, am_domainId)
+          .its('body')
+          .then((application: Application) => {
+            am_jwksTestApplication = application;
+            am_applicationSecret = application.settings.oauth.clientSecret;
+
+            const fakeJwksApi = ApiFakers.jwtApi({
+              publicKeyResolver: 'JWKS_URL',
+              resolverParameter: `${Cypress.env('am_gatewayServer')}/auth/${am_domainHrid}/oidc/.well-known/jwks.json`,
+              signature: 'RSA_RS256',
+            });
+
+            cy.log('-----  Create an API for JWKS test  -----');
+            importCreateApi(API_PUBLISHER_USER, fakeJwksApi)
+              .ok()
+              .its('body')
+              .then((createdApi) => {
+                jwksApi = createdApi;
+                publishPlan(API_PUBLISHER_USER, jwksApi.id, jwksApi.plans[0].id).ok();
+                deployApi(API_PUBLISHER_USER, jwksApi.id).ok();
+                startApi(API_PUBLISHER_USER, jwksApi.id).noContent();
+              });
+          });
+      });
+  });
+
+  after(() => {
+    cy.removeApi(hs256Api);
+    cy.removeApi(rs256Api);
+    cy.removeApi(noAuthPropApi);
```

Comment:
Maybe we should also delete AM stuff that we created in the global `before` ?
I see it deleted in JWKS test, but think it's more relevant here, cause there are created in the global init.
(Or move the AM stuff init to the before of the JWKS test case)

Reply:
Yes, true. 
Well, since I have the whole init in the global block it feels best to put the clean up for that into the global one as well.